### ISSUE TITLE
Correction form frontend

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/CORRECTIONS#184]
https://github.com/sanskrit-lexicon/CORRECTIONS/issues/122#issuecomment-161505496 says that someone posted a blank correction form, and we had to delete it from the history.
1. @funderburkjim it is time to put stricter validations in the Correction Form (maybe some javascript).

Right now it is 'required' to enter some details.
But if I enter an alphabet instead of numbers and period in the Lnumber field, it is accepted.
Blank or whitespaces / spaces / non-letters only should not be allowed.
1. Can't we autofill the Lnumber field and headword field also ? Right now the Dictname is being autofilled.

If I click on 'Corrections' from some entry page, this should autofill.
e.g.
![capture](https://cloud.githubusercontent.com/assets/5755132/11552754/3523d444-99ae-11e5-8edb-cebcf8557bc1.png)

Clicking here, I should get 
![capture](https://cloud.githubusercontent.com/assets/5755132/11552763/4e3c9eca-99ae-11e5-92ad-fe0559a25dfd.png)

**Comment by funderburkjim** [comment]
One problem you haven't considered is that some displays have multiple L-codes. So, there is no immediate way to fill in the L-code in such cases.

And, when the correction form is accessed via an Advanced search page, there is also no way to 
know which of the multiple 'headword' entries the user intends to correct.

The incorrect-form problem that prompted your suggestion occurs fairly infrequently.  
I didn't *have to* delete the line from cfr.tsv.

In summary, I don't think this is a serious enough problem to warrant attention now.

I'm adding the 'wontfix' label to this issue.
